lib/mercadolibre/core/users.rb

```diff
@@ -24,6 +24,13 @@ def get_user_accepted_payment_methods(user_id)
 
         results[:body].map { |r| Mercadolibre::Entity::PaymentMethod.new(r) }
       end
+
+      def revoke_access(user_id)
+        headers = { content_type: :json, accept: :json }
+        results = delete_request("/users/#{user_id}/applications/#{@app_key}?access_token=#{@access_token}", headers)
```

Comment:
This is like this because the ```RestClient.delete``` used on  ```delete_request```, takes only 2 arguments and in order to remove the access, we need to set the headers and also send the ```access_token``` as part of the params